src/main.rs

```diff
@@ -536,6 +569,13 @@ async fn consume_messenger_events(
             MessengerEvents::IncomingMessage(pubkey, onion_message) => {
                 onion_messenger.handle_onion_message(&pubkey, &onion_message)
             }
+            MessengerEvents::SendOutgoing => {
+                for peer in current_peers.keys() {
```

Comment:
This isn't going to have all of our peers in it - only the ones that we had on startup. 

Little annoying to have to double track (since the `OnionMessenger` also tracks online peers), but let's add/remove entries as we get connected / disconnected events so that the map is up to date.

Reply:
Ohhhhh good catch